presto-native-execution/presto_cpp/main/PeriodicTaskManager.h

```diff
@@ -57,12 +61,24 @@ class PeriodicTaskManager {
   void stop();
 
  private:
+  void scheduleExecutorStatsTask();
+  void scheduleTaskStatsTask();
+  void scheduleTaskCleanupTask();
+  void scheduleMemoryAllocatorStatsTask();
+  void schedulePrestoExchangeSourceMemoryStatsTask();
+  void scheduleAsyncDataCacheStatsTask();
+  void scheduleConnectorStatsTask();
+  void scheduleOperatingSystemStatsTask();
+
   folly::FunctionScheduler scheduler_;
   folly::CPUThreadPoolExecutor* driverCPUExecutor_;
   folly::IOThreadPoolExecutor* httpExecutor_;
   TaskManager* taskManager_;
-  velox::memory::MemoryManager& memoryManager_;
-  std::vector<std::shared_ptr<velox::connector::Connector>> connectors_;
+  velox::memory::MemoryAllocator* memoryAllocator_;
+  velox::cache::AsyncDataCache* asyncDataCache_;
+  const std::unordered_map<
```

Comment:
Put const member ahead of non-const member?

Reply:
Why is it non-const?

Reply:
I mean 
```
velox::memory::MemoryAllocator* const memoryAllocator_;
```
And the same for driverCPUExecutor_ etc which never change after set in ctor.